src/types.rs

```diff
@@ -103,6 +103,8 @@ pub struct Version {
     pub license: Option<String>,
     pub readme_path: Option<String>,
     pub links: VersionLinks,
+    pub crate_size: Option<u64>,
```

Comment:
Can you factor out the three new field additions into a separate commit?

Reply:
Ok. I'm redoing this in a new branch, starting with this commit.